R/one_minus_kaplan_meier.R

```diff
@@ -0,0 +1,60 @@
+#' One Minus Kaplan-Meier
+#'
+#' This funciton calculates the one minus Kaplan-Meier estimator of adverse events (while censoring all competing events)
+#' observed in `[0, tau]`.
+#' Please also refer to formula (4) in
+#'
+#' Stegherr R, Beyersmann J, Jehl V, Rufibach K, Leverkus F, Schmoor C, Friede T.
```

Comment:
please use the references syntax as e.g. in
https://github.com/openpharma/savvyr/blob/main/R/prop_trans_inc_dens_ce.R#L26